student.rb

```diff
@@ -0,0 +1,109 @@
+require 'csv'
+class Student
+    attr_accessor :name, :roll_number, :bengali, :english, :maths, :computer, :total, :percentage, :percentile
+    def initialize name, roll, bengali,english,maths,computer,t,per,p
+        @name=name
+        @roll_number = roll
+        @bengali = bengali
+        @english = english
+        @maths = maths
+        @computer = computer
+        @total=t
+        @percentage=per
+        @percentile=p
+    end
+
+    def calculate_total
+        @total = @bengali+@english+@maths+@computer
+    end
+    def calculate_percentage
+        @percentage=(@total/400.0)*100
+        p @percentage
+    end
+    def calculate_percentile
+        @percentile=(@percentage/10).floor
+    end
+    def to_s
+        "Name: #{@name}\nRoll Number: #{@roll_number}\nBengali: #{@bengali}\nMaths: #{@maths}
+        \nEnglish: #{@english}\nComputer: #{@computer}"
+    end
+end
+def store_file students
+    File.open("StudentGrade.txt","w") do |file|
+        students.each do |student|
+            file.puts "#{student.name},#{student.roll_number},#{student.bengali},#{student.maths},#{student.english},#{student.computer},#{student.total},#{student.percentage},#{student.percentile}"
+        end
```

Comment:
@qiashika , remove percentile from .txt file. Please, add percentile when you generate excel sheet only.